Using git as an Alternative to Robert's Rules.md

```diff
@@ -0,0 +1,68 @@
+
+* floor: A member is considered to “have the floor” when the chair calls upon him or her to speak. In most
+cases, it is improper to interrupt someone who currently has the floor.
+* order: Comments or discussion presented by a member are “in order” if the chair has properly
+recognized the member and the comments made pertain to the issue being discussed. Likewise, the
+member is “out of order” if either of these two conditions is not met.
+* quorum: The number of members needed to be present to conduct business legally. This number is set
+in your chapter’s bylaws.
+
+### Irrelevant
+Things which may fit into one of the categories above but which are obsoleted by this protocol.
+
+* voice vote: The chair requests that members indicate their preference on the issue by responding “yes”
+or “no” when asked to do so. The chair decides, based on the voices, whether the “yes” or “no” votes
+were in the majority. The chair may ask for a show of hands if it is unclear by voice vote how members
+voted. 
+
+# Proposal to replace Robert's Rules
+
+* ballot: A ballot is a pull request. Pull requests may be opened at any time by any member of the board or organization, similar to how Robert's Rules states that attendees may bring up business not on the agenda.
+* motion: A motion act of opening a pull request.
+* second: A second is a quorum of approvals of the pull request by the board.
+* floor: During board meetings, the floor is the right of a board member to speak. On a pull request, the floor is the right of board members and community members to engage in reasonable helpful discussion about an issue.
+
+## Pull Requests
+Pull requests serve as requests from a board member or the community to discuss an issue and make a decision. Pull requests can be opened at any time by any member of the community. Most pull requests will probably change policy or add files like minutes, resolutions, or guidelines for the community by modifying Markdown files stored in the [nonprofit-governance](https://github.com/nativesintech/nonprofit-governance) repository. Some changes, like adding information to the website, may be made in the form of code changes to other repositories.
+
+The board members and the community will discuss the issue publicly on the pull request in the form of requests for changes to the pull request, comments, and PR approvals.
+
+In the [nonprofit-governance](https://github.com/nativesintech/nonprofit-governance) repo, a vote is recorded by requiring all members of the Board group approve the PR. The measure is considered passed and any member of the board or the issuer of the pull request may merge the pull request and the issue is resolved.
+
+During board meetings, additional discussion can be made about each issue and board members may vote before the meeting or may wait until during the meeting to place their votes.
+
+Robert's Rules provides a mechanism for anonymous voting. Do we need this?
```

Comment:
I'm not sure we need to address this at this time given the size of the group that's engaged in decision making. I think we should defer and add something to address the need when it arises.

Reply:
Fixed.

Reply:
I didn't realize that RRO has anon voting but it makes sense. If we ever encounter a contentious issue it would be helpful to have.

Reply:
Is this something we would like to address now?

Reply:
I don't think we need it now. We can address/add it when we think it is needed.